code/game/machinery/vending.dm

```diff
@@ -659,8 +659,10 @@
 		flick(src.icon_vend,src)
 	playsound(src.loc, "sound/[vending_sound]", 100, 1)
 	spawn(src.vend_delay)
```

Comment:
Preferably this would use a timer rather than spawn, check out [timer documentation](https://github.com/Aurorastation/Aurora.3/wiki/Timers)